### DISCUSSION TITLE
Does this package support Large file uploads (chunking)?

### DISCUSSION THREAD

**Discussion by jcsoriano** [spatie/laravel-medialibrary#2646] [Q&A]
I'm looking for a way to handle large file uploads (e.g. 1+GB video files). Does this package support streaming / chunked uploads? I see https://github.com/spatie/laravel-medialibrary/issues/1107 but it was unanswered

**Comment by PaolaRuby** [comment]
Read the sources and find it out, I think it depends on the flysystem adapter

**Reply by jcsoriano** [reply]
Would you know which folder or file I should look into? Not familiar with the codebase of this package yet

**Reply by PaolaRuby** [reply]
I'm not the autor
https://github.com/spatie/laravel-medialibrary/tree/main/src
